current/text-analysis.md

```diff
@@ -0,0 +1,220 @@
+
+## Plain Text Extraction
+
+### All plain text
+
+This script extracts the crawl date, domain, URL, and plain text from HTML files in the sample ARC data (and saves the output to out/). By default, HTTP headers are included in the plain text that is extracted.
+
+```scala
+import io.archivesunleashed._
+import io.archivesunleashed.matchbox._
+
+RecordLoader.loadArchives("example.arc.gz", sc).keepValidPages()
+  .map(r => (r.getCrawlDate, r.getDomain, r.getUrl, RemoveHTML(r.getContentString)))
+  .saveAsTextFile("plain-text/")
+```
+
+If you wanted to use it on your own collection, you would change "src/test/resources/arc/example.arc.gz" to the directory with your own ARC or WARC files, and change "out/" on the last line to where you want to save your output data.
+
+Note that this will create a new directory to store the output, which cannot already exist.
```

Comment:
Maybe these generic examples around manipulating code should just go in one place at the beginning of the docs?